File: Examples/Examples/Demo/SuccessViewController.m
//
//  SuccessViewController.m
//  Examples
//
//  Created by Victor Zhu on 2021/5/13.
//  Copyright © 2020 Airwallex. All rights reserved.
//

#import "SuccessViewController.h"
#import "Airwallex/Core.h"

@interface SuccessViewController ()

@end

@implementation SuccessViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    self.view.backgroundColor = [AWXTheme sharedTheme].primaryBackgroundColor;

    UILabel *lab = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 200, 100)];
    lab.center = self.view.center;
    lab.text = @"Pay Result";
    lab.textColor = [AWXTheme sharedTheme].primaryTextColor;
    lab.font = [UIFont systemFontOfSize:30];
    lab.textAlignment = NSTextAlignmentCenter;
    [self.view addSubview:lab];
}

@end

File: Examples/Examples/Demo/SuccessViewController.swift
//
//  SuccessViewController.swift
//  Examples
//
//  Created by Weiping Li on 2025/2/12.
//  Copyright © 2025 Airwallex. All rights reserved.
//

import UIKit
import Airwallex

class SuccessViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        view.backgroundColor = .awxBackgroundPrimary

        let label = UILabel(frame: CGRect(x: 0, y: 0, width: 200, height: 100))
        label.center = view.center
        label.text = "Pay Result"
        label.textColor = .awxTextPrimary
        label.font = UIFont.systemFont(ofSize: 30)
        label.textAlignment = .center
        
        view.addSubview(label)
    }
}
